### DISCUSSION TITLE
Problemas para regatar os beneficios de estudante

### DISCUSSION THREAD

**Discussion by Vinitziin** [community/community#52639] [GitHub Education]
I am having trouble redeeming my student benefits. I have registered my institutional email on my account and have sent multiple images of my university identification card, but there is always some kind of error even though everything is correct. Could you please help me resolve this issue?

**Comment by samus-aran** [comment]
Hi @Vinitziin  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by Vinitziin** [reply]
hello, how are you, I can perform the rescue after a few days, apparently it was a technical problem
